EIPS/eip-epbs.md

```diff
@@ -0,0 +1,260 @@
+- Currently, validators have the time between receiving the full beacon block (including an execution payload) and the attesting deadline (4 seconds in Ethereum mainnet) to perform both consensus and execution state transition functions, check blob data availability and evaluate the new head of the blockchain. The remainder of the slot time is spent during less CPU-intense and critical task. By separating the validation of the execution and consensus part of the block, validators are only tasked to perform the consensus state transition function in this critical time before attesting, while execution and data availability validation is deferred for most of the remainder of the slot (between the builder's reveal time and the next attestation deadline). 
+- By removing the full execution payload size from the consensus block, it allows for faster network propagation on the critical path. 
+- It removes the increased reorg likeliness of including blob transactions in blocks. 
+- It prevents validators from missing attestations and strengthens the LMD weight properties of forkchoice in the event that builders produce invalid payloads. 
+- It removes the need of using middleware, not maintained by core developers, used by the vast majority of validators and it removes the need from client teams to continue supporting such middleware, improving thus coordination efforts for testing and deploying future hard forks. 
+
+## Specification
+
+### Execution Layer:
+
+No changes are required. 
+
+### Consensus Layer
+
+The full consensus changes can be found the consensus-specs Github repository. They are separated in 
+- [Beacon Chain](https://github.com/potuz/consensus-specs/blob/345e95504275d6c44b2bc251c50cb230fefb71b5/specs/_features/epbs/beacon-chain.md) changes
+- [Forkchoice](https://github.com/potuz/consensus-specs/blob/345e95504275d6c44b2bc251c50cb230fefb71b5/specs/_features/epbs/fork-choice.md) changes
+- [P2P](TODO-add-link) changes
+- [Honest validator guide](TODO-add-link) changes
+- A new [honest builder](https://github.com/potuz/consensus-specs/blob/345e95504275d6c44b2bc251c50cb230fefb71b5/specs/_features/epbs/builder.md) guide. 
+- [Fork logic](TODO-add-link) changes. 
+
+A summary of the main changes is included below, the [Rationale](#rationale) section contains explanation for most of the design decisions around these changes. 
+
+#### Constants
+
+| Name | Value | 
+| - | - | 
+| `PAYLOAD_ABSENT` | `uint8(0)` |
+| `PAYLOAD_PRESENT` | `uint8(1)` | 
+| `PAYLOAD_WITHHELD` | `uint8(2)` | 
+| `PAYLOAD_INVALID_STATUS` | `uint8(3)` |
```

Comment:
Why does this one have `_STATUS` appended and not the others?

Reply:
I'm horrible with names and this is a bit late to change, most names will change surely if this is accepted at implementation time, but names are a huge source of bikeshedding that I would want to avoid at this stage. The original reason is that `PAYLOAD_INVALID` already has a meaning in the spec repo. I would rather keep this one and let someone else when I'm not involved deal with name changing.

Reply:
sgtm!